dapps/react-dapp-v2/src/contexts/JsonRpcContext.tsx

```diff
@@ -252,9 +253,17 @@ export function JsonRpcContextProvider({
           },
         });
 
-        const valid = EthTransaction.fromSerializedTx(
-          signedTx as any
-        ).verifySignature();
+        let valid = false;
+        const [, reference] = chainId.split(":");
+        if (reference === "44787" || reference === "42220") {
```

Comment:
I realise the concrete chainIds aren't that simple to cross-reference the way they are specified here in the example dapp, but could we label these so they appear less like magic numbers? E.g.

```typescript
const CELO_ALFAJORES_CHAIN_ID = 44787

if (reference.toString() === CELO_ALFAJORES_CHAIN_ID) ...
```

Reply:
thanks for the suggestion, makes sense! i've updated and created celo variables :)